src/ngraph/node.cpp

```diff
@@ -46,32 +46,47 @@ Node::Node(const std::string& node_type, const NodeVector& arguments)
             m_inputs.emplace_back(this, i++, output);
         }
     }
+    set_output_size(output_size);
 }
 
-void Node::set_value_type_checked(const element::Type& element_type, const Shape& shape)
+static bool in_transition = true;
```

Comment:
I think this deserves a comment. Took me a bit to remember what it meant.

Reply:
ok